README.md

```diff
@@ -97,9 +99,14 @@
 * [Huobi (Futures)](https://www.huobi.com/)
 * [Okex (Futures)](https://www.okex.com/)
 
+### Decentralized Exchanges
+* [AirSwap](https://www.airswap.io/)
+* [AtomicDEX](https://atomicdex.io/)
+* [Kyberswap](https://kyberswap.com/swap)
+
```

Comment:
Good addition for the exchange section. But, as people are just starting to get into DEXs now, let's share with them a bit popular ones for now. Let's skip these exchanges for now as they are still in their early stages of development/adoption compared to uniswap/kybernetwork/idex.

Reply:
No worries. I've had trouble finding a more extensive DeFi / Dex list anywhere else, and thought this would be a good addition to the list. In terms of development, both Uniswap and Idex would be good ones to add that list.

- Of the three above, AirSwap would be the newest, and could probably be left out. 
- Kyberswap is the name of Kybernetwork's dex product (wasn't sure whether to link to the product itself, or just to Kybernetwork.
- AtomicDex is from Komodo, which has been around since 2014. Komodo has consistently put out some fantastic products, and actually processed the first AtomicSwap with Litecoin. I'd honestly say that between the 5, AtomicDex would probably be the most robust. 

**Next Steps:**
I'd vote for Including AtomicDEX, KyberNetwork, Idex, and Uniswap to be added that list.

Either that, or remove it entirely. It's up to you :)